public/js/components/Graph.js

```diff
@@ -11,29 +11,31 @@ export default class Graph extends React.Component {
     }
 
     componentDidMount() {
-        // TODO: Use actuial data
-        fetch(`keywords/trump`)
-            .then(res => res.json())
-            .then((dataPoints) => {
-                dataPoints = dataPoints
-                    .sort((a, b) => a.times - b.times)
+        fetch(`keywords/top`)
+            .then(res => {
+                return res.json()
+            })
+            .then(words => {
+                // for now graphing first word associated with the top article
+                // createing array of dataPoints
+                const dataPoints = words[0].votes
                     .map(point => [point.time, point.sum])
-                console.log(dataPoints)
+                // adding axis to dataPoints array
+				dataPoints.unshift(['Time', words[0].word])
                 this.setState({
-                    dataPoints
-                })
+					dataPoints: dataPoints
```

Comment:
I dont fully understand that operation yet so I don't use it. Can u tell me what its called so i can read up on it